hathor/merged_mining/coordinator.py

```diff
@@ -714,6 +732,40 @@ async def submit_to_bitcoin(self, job: SingleMinerJob, work: SingleMinerWork) ->
             # - unexpected-witness: transaction has inputs with witness but isn't marked as containing witnesses
             self.log.error('block rejected from Bitcoin', reason=res)
 
+    async def submit_to_digibyte(self, job: SingleMinerJob, work: SingleMinerWork) -> None:
+        """ Submit work to DigiByte RPC.
+        """
+        digibyte_rpc = self.coordinator.digibyte_rpc
+        assert digibyte_rpc is not None
+        # bitcoin_block = job.build_bitcoin_block(work)  # XXX: too expensive for now
+        bitcoin_block_header = job.build_parent_block_header(work)
+        block_hash = Hash(bitcoin_block_header.hash)
+        block_target = Target(int.from_bytes(bitcoin_block_header.bits, 'big'))
+        if block_hash.to_u256() > block_target.to_u256():
+            self.log.debug('high hash for DigiByte, keep mining')
+            return
+        if self.coordinator.should_skip_parent_submit(job.parent_height, ParentChain.DIGIBYTE):
+            self.log.debug('late winning share, skipping DigiByte submit')
+            return
+        bitcoin_block = job.build_bitcoin_block(work)  # XXX: far fewer cases, so it's OK now
```

Comment:
I'm a bit confused. Does DGB use the same structure as BTC?

Reply:
The block header is identical, there are some differences on the coinbase though, but that is built elsewhere. Also there might be differences with transactions, but it doesn't matter because they're also forwarded transparently.

Reply:
Got it.